nss-day-cohort-47/reactive-nutshell-the-fantastic-404s
# Articles correctly showing user and friends articles

# Description
Articles page now displays the logged in user's articles and the friends of that users articles.
## Type of change


# Testing Instructions
Fetch dcu-14-add-friends branch, go to the page for the app and sign in as mister@fantastic.com, click on the articles tab, see articles rendered. Friends articles will be styled differently.

# Checklist:
